idseq_dag/util/server.py

```diff
@@ -11,11 +11,15 @@
 
 
 MIN_INTERVAL_BETWEEN_DESCRIBE_INSTANCES = 180
-MAX_INTERVAL_BETWEEN_DESCRIBE_INSTANCES = 900
+MAX_INTERVAL_BETWEEN_DESCRIBE_INSTANCES = 900 # needs to match MAX_REFRESH_INTERVAL in idseq-web/app/jobs/autoscaling.py
```

Comment:
you could probably get it from the DAG json in that case, as both places have access to it